test/test_errors.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+import sys
+import os.path
+sys.path.append(os.path.join(os.path.abspath(os.path.dirname(__file__)), '..'))
+
+import constants
+
+def check(error, logname, with_hd, expected):
```

Comment:
I think it would be better to use a unit test framework for this (like `unittest`), especially as we get more unit tests over time.